addons/l10n_ug/data/account_tax_report_data.xml

```diff
@@ -514,21 +514,49 @@
                             <record id="F31_normal_tax" model="account.report.expression">
                                 <field name="label">hidden_computation_of_F31_using_normal_formula</field>
                                 <field name="engine">aggregation</field>
-                                <field name="formula">
-                                    (UG_TAX_D20.tax - UG_TAX_F30.disallowed)
-                                    * (UG_TAX_C1.base + UG_TAX_C2.base + UG_TAX_C4.base + UG_TAX_C5.base + UG_TAX_C6.base + UG_TAX_C7.base)
-                                    / (UG_TAX_C1.base + UG_TAX_C2.base + UG_TAX_C3.base + UG_TAX_C4.base + UG_TAX_C5.base + UG_TAX_C6.base + UG_TAX_C7.base)
```

Comment:
found it weird that UG_TAX_C3.base is not in the multiplication part ? Could you ask the person that did the localisation ?

Reply:
Ill ask but i think its correct like this because if the UG_TAX_C3 where to be in the multiplication part, then both terms are the same so it would be like multiplying by 10 and then dividing by 10 resulting in no change

Reply:
Yeah that's true, once you have the confirmation i will approve the task then :smile:

Reply:
aboo just confirmed to me that the formula is correct like this